### ISSUE TITLE
Opening some artists crashes MPD client

### ISSUE THREAD

**Issue by vojtapolasek** [lfcabend/mopidy-spotify-web#8]
Hi,
I really like your plugin, it provides features which are not provided by other mopidy spotify plugin. But I encounter a problem while accessing albums of some artists with GMPC MPD client. For example artist Yiruma. I try to list his albums through Spotify-web -> Your artists -> Yiruma, but GMPC always crashes. I attach the log created by Mopidy. The problem occurs near end before Mopidy is shut down.
[mopidy.txt](https://github.com/lfcabend/mopidy-spotify-web/files/688040/mopidy.txt)

**Comment by lfcabend** [comment]
I see some errors in log from a log statement.
like:

DEBUG    2017-01-05 18:34:55,527 [19637:SpotifyWebBackend-10] mopidy_spotify_web.library
  Going to add album Promeny
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mopidy/internal/log.py", line 176, in emit
    self.stream.write(message)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe1' in position 124: ordinal not in range(128)
Logged from file library.py, line 369

Looks like an UTF-8 encoding issue. Is this what you mean. I can have a look this weekend

**Comment by vojtapolasek** [comment]
Yes, I think this is what I mean. I am not sure if it causes the problem
but I can't see anything different.
